docs/managed-datahub/slack/saas-slack-troubleshoot.md

```diff
@@ -69,3 +69,24 @@ If you've subscribed to an entity, and then performed an action (i.e., raised an
 ### There is an issue with DataHub's systems
 If sending a test notification works, and you've verified that none of the above cases apply, then you should contact your DataHub Customer Success rep to help troubleshoot and resolve the issue. 
 
+## Slack bot issues
+This is to troubleshoot issues with your slack bot
+
+### Command failed with error "dispatch_failed"
+If you've installed the slack bot, but your commands are failing with an error 'dispatch_failed', you can try the following to correct it.
+1. Open your acryl instance with the following url: `<your-instance-base-url>/settings/integrations/slack?display_all_configs=true`.
+2. Switch to the **Bot Token** tab.
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/slack/verify_tokens_1.png"/>
+</p>
+3. The values will be `**`'d out, but if any of these fields are empty (ie. Signing Secret), then you'll want to fill them in.
```

Comment:
You might need to put a linebreak after `</p>` & before #3 -- the rich diff isn't rendering the markdown/handling linebreaks accurately from here on out: 
![CleanShot 2024-10-28 at 11 31 40@2x](https://github.com/user-attachments/assets/f10fff29-1d2a-4f67-9279-261091d23935)
![CleanShot 2024-10-28 at 11 31 52@2x](https://github.com/user-attachments/assets/a815e876-04f6-4489-b550-32bdd290d97e)

Reply:
Ah weird. It looks okay when built though:
<img width="996" alt="image" src="https://github.com/user-attachments/assets/fb4691a5-b867-4a51-ac8e-5b9d0bae687f">
The linebreak adds extra space in the build